### PR TITLE
Enable .NET CI

### PR DESCRIPTION
**Pull request by lucasmellos**

### CODE REVIEW

File: .github/actions/docker-compose/action.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+name: Run Docker compose file
+description: Buld and push image to SignalWire registry
```
Comment (Null-Is-Null) [inline .github/actions/docker-compose/action.yml:2]: Should this description be changed?

File: .github/actions/sonarqube/action.yml [hunk 2]
```diff
@@ -1,11 +1,53 @@
...
 description: Run code coverage analysys
 
+inputs:
+  LANGUAGE:
+    required: true
+    
 runs:
   using: "composite"
   steps:
+    - name: Set up JDK 11
```
Comment (Null-Is-Null) [inline .github/actions/sonarqube/action.yml:11]: Should this be .Net instead of JDK 11?

**Review by Null-Is-Null — APPROVED** [review]
Looks good. Had a couple comments but not on anything that would be breaking.

File: .github/actions/sonarqube/action.yml (hunk 2 above)
Comment (lucasmellos) [reply .github/actions/sonarqube/action.yml:11]: nope, sonarqube is java based

**Pull request merged**